cypress/fixtures/projects.json

```diff
@@ -3,7 +3,7 @@
     "id": "6d5fff43-d691-445d-a41a-7d0c639080e6",
     "numIssues": 73,
     "numEvents24h": 7,
-    "status": "error",
+    "status": "critical",
```

Comment:
I wouldn't change these as they should reflect the API data.

Reply:
Sounds good, I reverted them to their original value.